tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -59,27 +59,18 @@ build_rootfs() {
 	fi
 	# trim whitespace
 	PACKAGES=$(echo $PACKAGES |xargs )
-	EXTRA_PKGS=$(echo $EXTRA_PKGS |xargs)
 	# add comma as debootstrap needs , separated package names.
 	# Don't change $PACKAGES in config.sh to include ','
 	# This is done to maintain consistency
 	PACKAGES=$(echo $PACKAGES | sed  -e 's/ /,/g' )
-	EXTRA_PKGS=$(echo $EXTRA_PKGS | sed  -e 's/ /,/g' )
-
-	# extra packages are added to packages and finally passed to debootstrap
-	if [ "${EXTRA_PKGS}" = ""  ]; then
-		echo "no extra packages"
-	else
-		PACKAGES="${PACKAGES},${EXTRA_PKGS}"
-	fi
 
 	${PKG_MANAGER} --variant=minbase \
 		--arch=${ARCHITECTURE}\
 		--include="$PACKAGES" \
 		${OS_NAME} \
 		${ROOTFS_DIR}
 
-	chroot $ROOTFS_DIR ln -s /lib/systemd/systemd /usr/lib/systemd/systemd
```

Comment:
@Jakob-Naucke Why is this no longer required?

Reply:
On Ubuntu 20.04, those two files are already present. They're identical (no soft/hard links).

Reply:
Ok. Thanks for clarifying.